.github/workflows/deploy.yaml

```diff
@@ -462,6 +462,24 @@ jobs:
           REGISTRATION_SERVICE_VERSION: 0.0.1-SNAPSHOT
           DATASPACE_DID_HOST: ${{ needs.Deploy-Dataspace.outputs.dataspace_did_host }}
 
+      - name: 'Wait for participant registration'
+        run: |
+          
+          hostDid="did:web:$DATASPACE_DID_HOST"
```

Comment:
`host` is somewhat unexpected here, since the other variable is also a host.

Consider inlining those two

Reply:
Agreed! Will change.